Dockerfile-custom

```diff
@@ -0,0 +1,9 @@
+FROM localhost/maestro-base:latest
+
+# COPY <src> agent.yaml      # copy agent file into image
+# COPY <dest> workflow.yaml  # copy workflow file into image
+
+CMD ["serve", "agent.yaml", "workflow.yaml", "--port=5000"]  # for workflow serve
+#CMD ["serve", "agent.yaml", "--port=5000"]                  # for agent serve
```

Comment:
why change the port isn't the default 8000 correct based on the base image? 

also I found in my previous work that when running from docker we need to set `"--host", "0.0.0.0"` as well to handle outside connections